src/KubeConfigModels/UserCredentials.cs

```diff
@@ -8,9 +9,15 @@ public class UserCredentials
         [YamlMember(Alias = "client-certificate-data")]
         public string ClientCertificateData { get; set; }
 
+        [YamlMember(Alias = "client-certificate")]
+        public string ClientCertificate { get; set; }
+
         [YamlMember(Alias = "client-key-data")]
         public string ClientKeyData { get; set; }
 
+        [YamlMember(Alias = "client-key")]
+        public string ClientKey { get; set; }

```

Comment:
Same as client-certificate. Thought client-key-data stored a path, but they are the certificate itself. Now several tests are broken.